src/application.coffee

```diff
@@ -332,6 +363,122 @@ updateStatus =
 	forceNext: false
 	intervalHandle: null
 
+updateStrategies =
```

Comment:
Yup, the thing is what changes is mainly the order in which things and done... but true, something to keep in mind and possibly DRY out if we add more strategies in the future...